README.md

```diff
@@ -3,80 +3,101 @@
+help open-source sustainability.
+
+Naturally, we're dogfooding SourceCred on the SourceCred
+project itself. Most of our work happens on GitHub, so we import our GitHub data
+into a graph. Pulls and commits and users are all nodes, with edges connecting
+pulls to their authors, pulls to their commits, and so forth. Thus, if Pearl
+writes a bug report, Steven writes a pull request referencing that bug report, and
+Amethyst reviews and merges the pull, then Pearl, Steven, and Amethyst will all
+earn some cred in the project.
+
+Some contributions will be hard to spot on GitHub. For example, someone might
+help organize a SourceCred meetup, or work on community management. To make
+sure those contributors get the cred they deserve, we're creating a Spotlight
+plugin. The Spotlight allows contributors to draw attention to contributions
+that might otherwise be overlooked.  The more people that endorse the
+spotlight, the more cred it's worth.
+
+#### PageRank
+
+SourceCred uses a modified [PageRank] algorithm for cred attribution. Using
+PageRank is important, as it means cred attribution is not just a popularity
+contest, and is robust to sockpuppetry.
+
+For example, if 100 sockpuppet accounts spotlight someone, they will not earn
+much cred, as the sockpuppets themselves have very little cred. Conversely, if
+core maintainers spotlight someone, they will earn a lot more cred.
+
+If you're interested in learning more about our modified PageRank, check out
+[this issue].
+
 [PageRank]: https://en.wikipedia.org/wiki/PageRank
+[this issue]: https://github.com/sourcecred/sourcecred/issues/476
 
-## Principles
 
-SourceCred aims to be:
+#### Cred and Grain
 
-1. **Transparent**
+As a reputation score, a contributor's cred can continually fluctuate. For
+example, if someone writes an exciting new feature, they might earn a lot of
+cred; if that feature turns out to be full of bugs and unmaintainable, the cred
+might decay pretty quickly.
 
-   If it's to be a legitimate and accepted way of tracking credit in projects,
-   cred attribution can't be a black-box. SourceCred will provide tools that
-   make it easy to dive into the cred attribution, and see exactly why
-   contributions were valued the way they were.
+We think projects will want to have an accumulating token that represents
+someone's aggregated support for a project over time. This token is called
+grain. Projects that turn on grain can "mint" new grain every week and issue it
+to contributors in proportion to their cred.
 
-2. **Community-controlled**
+In the future, it will be possible to turn a project's grain into a
+transferrable cryptoasset (likely an [ERC20 token] on [Ethereum]). At that
+point, grain can be seen as a "proof of patronage" of the project. For example,
+people who have SourceCred grain have either directly contributed to SourceCred
+or bought the grain from someone who did, and thus are "patrons" of the
```

Comment:
This is of course not true (it’s a transitive relation). It’d be more
precise to say something like,

> A patron of a project is someone who has either contributed to a
> project directly, or purchased grain from a patron of the project.

I’m open to leaving the “white lie” in for ease of readability, but also
I don’t think that the above is too bad. What do you think?

Reply:
Good catch. I'd prefer to be precise, including sloppy white lies is a bad habit and will erode credibility in the eyes of people who are paying close attention.

Reply:
Great. I agree, and think that this is the right call.